models/pull.go

```diff
@@ -636,3 +636,19 @@ func (pr *PullRequest) updateCommitDivergence(e Engine, ahead, behind int) error
 func (pr *PullRequest) IsSameRepo() bool {
 	return pr.BaseRepoID == pr.HeadRepoID
 }
+
+// GetPullRequestByHeadBranch returns a pr by head branch
+func GetPullRequestByHeadBranch(headBranch string, headRepo *Repository) (pr *PullRequest, err error) {
```

Comment:
you can have multible pulls with the same headbranch - how do you handle this?

Reply:
Hmmm I don't. I guess I'll change that.

Reply:
Fixed, I now check all prs for a head branch